packages/app-builder/src/components/Scenario/ScenarioValidationError.tsx

```diff
@@ -1,47 +1,25 @@
-import { type EvaluationError } from '@app-builder/models';
-import {
-  adaptEvaluationErrorViewModels,
-  useGetNodeEvaluationErrorMessage,
-} from '@app-builder/services/validation';
 import clsx from 'clsx';
-import type React from 'react';
-
-export function ScenarioValidationError({
-  className,
-  children,
-}: {
-  className?: string;
-  children?: React.ReactNode;
-}) {
-  return (
-    <div
-      className={clsx(
-        'bg-red-05 text-s flex h-8 items-center justify-center rounded px-2 py-1 font-medium text-red-100',
-        className,
-      )}
-    >
-      {children}
-    </div>
-  );
-}
 
 export function EvaluationErrors({
-  evaluationErrors,
+  errors,
   className,
 }: {
-  evaluationErrors: EvaluationError[];
+  errors: string[];
```

Comment:
`EvaluationErrors` should be error formating agnostic (required to handle specific root error formatter)